src/plugins/controllerboard2x/index.js

```diff
@@ -1,70 +1,88 @@
-(function () {
-  var sharedFunctions = require('./public/js/telemetryToSystemPower.js');
-  var PREFERENCES_NS = 'plugins';
-  var SystemEnvionment = function SystemEnvironment(name, deps) {
-    console.log('Controllerboard2x:SystemEnvironment plugin loaded');
-    deps.globalEventLoop.on('mcu.status', function (data) {
-    });
-  };
-  var SystemPower = function SystemPower(name, deps) {
-    console.log('Controllerboard2x:SystemPower plugin loaded');
-    this.config = deps.config;
-    this.cockpit = deps.cockpit;
-    var self = this;
-    deps.globalEventLoop.on('mcu.status', function (data) {
-      var mappedPowerObject = sharedFunctions.telemetryToSystemPower(data);
-    });
-    deps.cockpit.on('plugin.systemPower.powerESCs', function (enable) {
-      if (enable){
-        deps.globalEventLoop.emit('mcu.SendCommand', 'escp(1)');
-      } else {
-        deps.globalEventLoop.emit('mcu.SendCommand', 'escp(0)');      
-      }
-    });
+var sharedFunctions = require('./public/js/telemetryToSystemPower.js');
+var PREFERENCES_NS = 'plugins';
 
-  };
-  SystemPower.prototype.getSettingSchema = function getSettingSchema() {
-    //Technically the json-editor supports "watch" that can be used
-    //to dynamically add items from an array as an ENUM to a select
-    //in the same document.  Have an issue filed as I cannot get it
-    //to work.  WORKAROUND: Manually parse the config for the Battery
-    //types and inject them in to the schema.  TODO: signal an update
-    //to the schema that is caches in several spots when a Battery
-    //is changed.
-    BatteryOptions = [
-      'TrustFire',
-      'LiFePO4'
-    ];
-    var b = this.config.preferences.get(PREFERENCES_NS);
```

Comment:
This was causing the double listing of selectable batteries. I removed the loading from the config file and made the battery profiles statically selectable here in the plugin.